.changeset/rare-squids-begin.md

```diff
@@ -0,0 +1,26 @@
+---
+"wrangler": patch
+---
+
+feat: implement time travel for experimental d1 dbs
+
+This PR adds two commands under `wrangler d1 time-travel`:
+
+```
+Interact with D1 Time Travel
+
+Commands:
+
+wrangler d1 time-travel info <database> Get information about D1 at a point in time.
+Options:
+      --timestamp  timestamp (accepts unix timestamp or ISO strings) to use for time travel  [string]
```

Comment:
- ISO should be (more clearly) RFC3339 - the latter is more clearly specified (see: https://ijmacd.github.io/rfc3339-iso8601/)
- The help string should show an example RFC3339 timestamp - e.g. 

```
accepts a Unix (seconds from epoch) or RFC3339 timestamp (e.g. 2023-07-09T17:31:11+00:00) to retrieve a bookmark for.
```